test/benchmarks/driverBench/index.js

```diff
@@ -1,338 +1,42 @@
 'use strict';
 
 const MongoBench = require('../mongoBench');
-const { writeFile } = require('fs/promises');
 
 const Runner = MongoBench.Runner;
-const commonHelpers = require('./common');
 
 let BSON = require('bson');
 
-try {
-  BSON = require('bson-ext');
-} catch (_) {
-  // do not care
-}
+let bsonType = 'js-bson';
```

Comment:
not sure if this `let` will trigger our lint now, but otherwise lgtm

Reply:
Lint passed.